AsyncDisplayKit/ASTextNode.mm

```diff
@@ -366,31 +366,35 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
 
 - (void)setAttributedText:(NSAttributedString *)attributedText
 {
-  std::lock_guard<std::recursive_mutex> l(_textLock);
   
   if (attributedText == nil) {
     attributedText = [[NSAttributedString alloc] initWithString:@"" attributes:nil];
   }
+  
+  // Don't hold textLock for too long.
```

Comment:
@Adlai-Holler this probably only provides value if we call out that we're interested in avoiding locking the displaynode lock while holding this one.